test/test_io.py

```diff
@@ -74,7 +75,7 @@ class TestIO(unittest.TestCase):
     def test_write_read_video(self):
         with temp_video(10, 300, 300, 5, lossless=True) as (f_name, data):
             lv, _, info = io.read_video(f_name)
-            self.assertTrue(data.equal(lv))
+            assert_equal(data, lv)
```

Comment:
I think data and lv are floating tensors, so I'm not sure whether `assert_close` would make more sense here.
Regardless, this isn't a regression w.r.t master so as long as the tests pass, this is probably fine